src/main/webapp/app/course/manage/detail/course-detail.component.html

```diff
@@ -3,14 +3,16 @@
         <div class="course-info-bar" [ngStyle]="{ backgroundColor: course.color || ARTEMIS_DEFAULT_COLOR }" id="course-detail-info-bar">
             <div class="row">
                 <div class="col general-info">
-                    <h3 class="fw-medium">{{ course.title }}</h3>
-                    <h4 class="fw-medium">{{ courseDescription || ('artemisApp.courseOverview.noDescription' | artemisTranslate) }}</h4>
-                    <h6 class="fw-medium" role="button" *ngIf="enableShowMore && !longDescriptionShown" (click)="toggleCourseDescription()">
-                        {{ 'artemisApp.courseOverview.showMore' | artemisTranslate }}
-                    </h6>
-                    <h6 class="fw-medium" role="button" *ngIf="enableShowMore && longDescriptionShown" (click)="toggleCourseDescription()">
-                        {{ 'artemisApp.courseOverview.showLess' | artemisTranslate }}
-                    </h6>
+                    <h3 class="fw-medium" [ngStyle]="courseDescription ? {} : { 'margin-bottom': 0 }">{{ course.title }}</h3>
```

Comment:
```suggestion
                    <h3 class="fw-medium" [class.mb-0]="!courseDescription">{{ course.title }}</h3>
```
Should also work (please test this first :D) ->  https://angular.io/guide/class-binding